src/integration-test/java/developerTC/TestAssetRegistration_08.java

```diff
@@ -186,4 +189,48 @@ public void testGetDIDAllTypes() throws IOException {
         Assertions.assertEquals(remoteAgent.getDID().getID(), remoteOperationAsset.getDID().getID());
     }
 
+    @Test
+    public void testProvenaceForNamedAsset01()  {
+
+
+
+        Map<String,Object> additionalMeta = new HashMap<>();
+        additionalMeta.put("name","BMW data ");
+        DataAsset  resourceAsset = ResourceAsset.create("assets/test_content.json",additionalMeta);
+
+        //  include Provenace
+        resourceAsset= resourceAsset.includeProvenace();
+
+        Map<String, Object> provData = JSON.toMap(resourceAsset.getMetadata().get("provenance").toString());
+        assertTrue(provData.get("wasAssociatedWith").toString().contains("BMW data "));
```

Comment:
Why do we expect the string to be in the wasAccociatedWith block? Can we have some better description of what this is testing?

Reply:
Reverted the changes, will raise PR separately for a separate issue.